app/uk/gov/hmrc/cdsreimbursementclaim/models/claim/CompleteClaim.scala

```diff
@@ -56,7 +55,7 @@ object CompleteClaim {
     maybeBasisOfClaimAnswer: Option[CompleteBasisOfClaimAnswer],
     maybeCompleteBankAccountDetailAnswer: Option[CompleteBankAccountDetailAnswer],
     supportingEvidenceAnswer: SupportingEvidenceAnswer,
-    completeCommodityDetailsAnswer: CompleteCommodityDetailsAnswer,
+    commodityDetailsAnswer: CommodityDetails,
```

Comment:
This should be renamed because it's now dealing with reasons for claim not the commodity details